highfidelity/hifi
# Commerce: More security images!

Commerce: Now with 34 security images!

**Test Plan:**
1. Create a new account, log in to that account, and open the Wallet app
2. Click the "Set Up Wallet" button
3. Verify that you see six random security images, none of which are duplicates
4. Click the "Back" button, then click "Set Up Wallet" again.
5. Verify that you see six new random security images (i.e. not the same set of six as in (3)), none of which are duplicates
6. Continue through the wallet setup process, ensuring that you see your selected security image throughout
7. After setting up your wallet (it doesn't matter if you're seeded or not), restart Interface
8. Open the Wallet app. Verify that you see your selected security image on the authentication page.
9. Go to the "Security" tab
10. Next to "Security Pic", click "Change"
11. Verify that you see six random security images, none of which are duplicates
12. Click "Cancel", then click "Change" next to "Security Pic" again
13. Verify that you see six new random security images (i.e. not the same set of six as in (11)), none of which are duplicates
14. Select a new security image and ensure that the new one appears throughout the Wallet
